extensions/ql-vscode/src/extension.ts

```diff
@@ -733,6 +743,22 @@ async function activateWithInstalledDistribution(
         cancellable: true
       })
   );
+
+  ctx.subscriptions.push(
+    commandRunnerWithProgress(
+      'codeQL.codeLensQuickEval',
+      async (
+        progress: ProgressCallback,
+        token: CancellationToken,
+        uri: Uri | undefined,
```

Comment:
I don't think this can ever be `undefined`, so might as well make the type strict.